languages/go/docs/style-guide.md

```diff
@@ -0,0 +1,72 @@
+# Style Guide
+
+## Stub functions
+
+Stub functions best contain a `panic` containing the phrase: `Please implement the XY function`.
+
+Example:
+```go
+// Message extracts the message from the provided log line.
+func Message(line string) string {
+	panic("Please implement the Message function")
```

Comment:
massive nitpick from me, but should panic messages start lower case, as with [error messages](https://github.com/golang/go/wiki/CodeReviewComments#error-strings)?

Reply:
Good question... I would have done the same as you suggest. They were capitalised in the experiment: https://github.com/exercism/research_experiment_1/blob/master/exercises/go-1-a/go_1_a.go#L5

I think better remove the capitalising!